
string connect_name 		#
bool   is_connected 	#
